docs/resources/l3out_consumer_label.md

```diff
@@ -58,6 +58,19 @@ resource "aci_l3out_consumer_label" "full_example_l3_outside" {
   owner_key   = "owner_key_1"
   owner_tag   = "owner_tag_1"
   tag         = "lemon-chiffon"
+  relation_from_l3out_consumer_label_to_external_epgs = [
+    {
+      annotation = "annotation_1"
+      target_dn  = "uni/tn-test_tenant/out-test_l3_outside/instP-inst_profile"
```

Comment:
can these be references instead of static dn?

Reply:
Gaspard changes will fix the static DN on the doc files.